notes/programming-languages/python/packages/pandas.md

```diff
@@ -0,0 +1,123 @@
+# Python Language Overview
+
+### The `pandas` Module
```

Comment:
I'd prefer to use the term "Package" to refer to third-party open source packages downloadable via pip, and reserve the term "Module" for built-in Python modules which don't require additional installation.